sycl/doc/design/spirv-extensions/SPV_INTEL_joint_matrix.asciidoc

```diff
@@ -270,11 +306,19 @@ where elements of the matrix must be stored and arranged according to 'Layout'.
 'Stride' is the number of elements in memory between beginnings of successive
 rows, columns (or words) of the 'Object'. It must be a scalar integer type. +
  +
-'Layout' indicates how the values stored to memory are arranged. It must be the
-result of a constant instruction. +
+'Layout' indicates how the values stored to memory are arranged.
```

Comment:
Similar to the load instruction, I would find this sentence to be clearer if it were changed to say that it indicates how the values that are stored are arranged **in memory** (assuming this is the intent).

Reply:
Applied